src/resolver.js

```diff
@@ -12,13 +12,13 @@ const util = require('./util')
  * @param {Uint8Array} binaryBlob - Binary representation of a CBOR block
  * @param {string} [path='/'] - Path that should be resolved
  */
-exports.resolve = (binaryBlob, path) => {
+exports.resolve = (binaryBlob, path = '/') => {
```

Comment:
I guess this is technically correct, and the docs say it's so, but in practice this is `'/'.split('/').filter(Boolean)` === `[]`, which is a weird thing to do! Not a big deal, we could probably save a cycle or two with just `''` tho.

Reply:
Done!